docs/core/development/review-process.rst

```diff
@@ -0,0 +1,248 @@
+Twisted Code Review Process
+===========================
+
+
+All commits to Twisted's trunk must follow this review process.
+The only exception is for the [https://github.com/twisted/twisted/blob/trunk/docs/fun/Twisted.Quotes Twisted Quotes] file.
+There are no other exceptions!
```

Comment:
The quotes file is pretty much dead at this point. We no longer select a QOTR, so I think we might as well get rid of this caveat.

```suggestion
```